README

```diff
@@ -230,6 +230,18 @@ SUSE:
 Alternatively the MPI requirement can be met by using OpenHPC
 packages.
 
+If power governing or power balancing is the intended use case
+for GEOPM deployment then there is an additional dependency
+on the system being configured to support RAPL execute (on a compute node):
+    ./tutorial/admin/00_test_prereqs.sh
+
+If the script output contains:
+    WARNING: The lock bit for the PKG_POWER_LIMIT MSR is set.  The power_balancer
+             and power_governor agents will not function properly until this is cleared.
+
+Please enable RAPL in your BIOS and if such an option does't exist please
+contact your BIOS vendor to obtain a RAPL supported BIOS.
+
```

Comment:
I'd add one more line that says something like for additional information, please contact the GEOPM team.